README.md

```diff
@@ -87,5 +88,11 @@ curl -v -H "Host: sklearn-iris.default.example.com" http://localhost:8080/v1/mod
 
 * Follow the [example here](docs/samples/client/kfserving_sdk_sample.ipynb) to use the KFServing SDK to create, rollout, promote, and delete an InferenceService instance.
 
-### Contribute
-* [Developer Guide](/docs/DEVELOPER_GUIDE.md).
+### KFServing Concepts and Data Plane
+[KFServing Concepts and Data Plane](./docs/README.md)
+
+### KFServing API Reference
+[KFServing API Docs](./docs/apis/README.md)
+
+### Developer and Contributor Guide
+If you are a developer and want to run your own development builds, or contribute to KFServing, follow the [Developer Guide](/docs/DEVELOPER_GUIDE.md).
```

Comment:
Unnecessary verbiage i think

Reply:
we should link contributor guide(https://github.com/kubeflow/kfserving/blob/master/CONTRIBUTING.md) also then.

Reply:
a lot of people do not know how to sign the CLA, sometimes they sign the Kubernetes one instead of google CLA.

Reply:
added link to contributors guide as well

Reply:
done